src/components/messenger/list/index.tsx

```diff
@@ -221,28 +222,42 @@ export class Container extends React.Component<Properties, State> {
     );
   }
 
+  renderSettingsMenu() {
+    return (
+      <SettingsMenu
+        onLogout={this.props.logout}
+        userName={this.props.userName}
+        userHandle={this.props.userHandle}
+        userAvatarUrl={this.props.userAvatarUrl}
+      />
+    );
+  }
+
   render() {
     return (
       <>
         {this.props.includeTitleBar && this.renderTitleBar()}
 
         {this.props.stage === SagaStage.None && (
-          <RewardsBar
-            zeroPreviousDay={this.props.zeroPreviousDay}
-            isRewardsLoading={this.props.isRewardsLoading}
-            isMessengerFullScreen={this.props.isMessengerFullScreen}
-            showRewardsInTooltip={this.props.showRewardsInTooltip}
-            showRewardsInPopup={this.props.showRewardsInPopup}
-            isFirstTimeLogin={this.props.isFirstTimeLogin}
-            includeRewardsAvatar={this.props.includeRewardsAvatar}
-            userName={this.props.userName}
-            userHandle={this.props.userHandle}
-            userAvatarUrl={this.props.userAvatarUrl}
-            onRewardsPopupClose={this.props.rewardsPopupClosed}
-            onRewardsTooltipClose={this.props.rewardsTooltipClosed}
-            onLogout={this.props.logout}
-            hasLoadedConversation={this.props?.conversations[0]?.hasLoadedMessages}
-          />
+          <div {...cnMessageList('profile-bar')}>
+            {this.props.includeRewardsAvatar && (
```

Comment:
Probably not a "rewardsAvatar" any more?

Reply:
Been thinking about renaming a few things in this PR, will have a quick look tomorrow.  `RewardsBar` should probably be renamed, it's more of a button with popups.

Reply:
Yeah, the next phase of refactoring can happen as follow ups. This is good enough to allow the story to be completed at least.